recipes/ovs_offload/1_virt_ovs_vxlan_up_down.py

```diff
@@ -0,0 +1,71 @@
+"""
+This test verifies proper handling of up/down events on bridge device.
+Test generates traffic from guest using ping and checks that no errors
+reported on dmesg after each up/down event.
+Created for fix of issue 1048140.
+"""
+
+from lnst.Controller.Task import ctl
+from lnst.RecipeCommon.ModuleWrap import ping
+from Testlib import Testlib
+from lnst.Common.ExecCmd import exec_cmd
+
+
+# ------
+# SETUP
+# ------
+
+tl = Testlib(ctl)
+
+# hosts
+host1 = ctl.get_host("h1")
+host2 = ctl.get_host("h2")
+
+# guest machines
+guest1 = ctl.get_host("test_host1")
+
+guest1.sync_resources(modules=["IcmpPing"])
+
+# ------
+# TESTS
+# ------
+
+g1_nic = guest1.get_interface("if1")
+h2_nic = host2.get_device("int0")
+
+bridge_dev = host1.get_interface("br-pys")
+total_time_sec = 60
+
+
+def start_pings(seconds):
+    ping_opts = {"count": seconds, "interval": 1.0, "limit_rate": 40}
+    return ping((guest1, g1_nic, 0, {"scope": 0}),
+                (host2, h2_nic, 0, {"scope": 0}),
+                options=ping_opts, expect="pass", bg=True)
+
+
+def check_dmesg():
+    cmd = "dmesg | tail -n 100"
+    data_stdout = exec_cmd(cmd)[0]
```

Comment:
aren't you reading here dmesg from the lnst controller?
you need to read from the hypervisor